showcase/testem.js

```diff
@@ -6,6 +6,7 @@
 'use strict';
 
 module.exports = {
+  parallel: process.env.CI ? -1 : 1,
```

Comment:
for posterity, setting parallel execution to `-1` means auto-detect and use the maximum number of parallel processes possible, while setting it to 1 means executing tests serially, one after the other

Reply:
add a comment